### DISCUSSION TITLE
Fullscreen Windows

### DISCUSSION THREAD

**Discussion by jbjoeB** [ianyh/Amethyst#1376] [Q&A]
Hello,

Just found this app today and love it.. i'm coming from linux just over two days now and i had to get something tiling because i'm used to i3, bspwm, dwm etc..

Question is Why are my windows always in fullscreen mode? Is it because Amethyst is always on and its waiting to tile windows or is there something i'm doing wrong that i'm overlooking.. Would like to know

Thanks

**Comment by ianyh** [comment]
I'm not sure what you mean by full screen. If you mean the windows all tiled to be full screen than you may be using the fullscreen layout. You can cycle through layouts with option + shift + space by default.

**Reply by jbjoeB** [reply]
Thanks for the reply..

I'm tripping, i just realized that i was using a 15' inch screen and when i would tiled it would take up the whole screen now im on a 27' monitor and its taking up the screen. it would do that with 2 apps. i'm not used to this tiling on such a wide monitor

Thanks again.

**Reply by ianyh** [reply]
So to clarify, is there an issue here that needs debugging?
